### PR TITLE
Add "Buy me Coffee" Integration

### PR DESCRIPTION
**Pull request by rribeiro1**
## Feature

[Buy me a coffee](https://www.buymeacoffee.com/) gives the audience of the portfolio an easy way to say thanks by "buying you a coffee".

After enabling it, it will show on every page of the portfolio.

## Configuration
```yaml
buymeacoffee:
  enabled: true
  username: your_username
  color: "#007bff"
  message: "Thank you for visiting."                    # Message to be displayed aside from the button for a few seconds.
  description: "Support me on Buy me a coffee!"
```

## How it looks like:
Screenshot 2022-03-13 at 21 17 00

Screenshot 2022-03-13 at 21 16 52

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -16,12 +16,14 @@
...
   </main>
-
   {% include footer.html %}
 
   {% include analytics.html %}
   {% include scripts.html %}
 
+  {%- if  site.author.buymeacoffee -%}
+    <script data-name="BMC-Widget" data-cfasync="false" src="https://cdnjs.buymeacoffee.com/1.0.0/widget.prod.min.js" data-id="{{ site.author.buymeacoffee }}" data-description="Support me on Buy me a coffee!" data-message="" data-color="#FFDD00" data-position="Right" data-x_margin="16" data-y_margin="16"></script>
+  {%- endif -%}
```
Comment (rribeiro1) [inline _layouts/default.html:26]: This button script was provided by the Buy me a coffee website, it's a default configuration, but we might want to let `data-color` be configurable as well as `data-description`.

File: _sass/_footer.scss [hunk 2]
```diff
@@ -15,7 +15,7 @@
     writing-mode: vertical-rl;
     transform: rotate(180deg);
     position: fixed;
-    bottom: 56px;
+    bottom: 120px;
```
Comment (rribeiro1) [inline _sass/_footer.scss:18]: Lifting it a bit more not to overlay the button.

File: _layouts/default.html (hunk 1 above)
Comment (YoussefRaafatNasry) [reply _layouts/default.html:26]: Can we do that in this PR?
We can create a key in `config.yml` with these properties.

**Review by YoussefRaafatNasry — CHANGES_REQUESTED** [review]

File: _layouts/default.html (hunk 1 above)
Comment (rribeiro1) [reply _layouts/default.html:26]: Done.

File: docs/_config.yml [hunk 3]
```diff
@@ -73,6 +73,16 @@ analytics:
...
 
 
+### Buy me a coffee ###
+# See https://www.buymeacoffee.com/
+buymeacoffee:
+  enabled: false
+  username: your_username
+  color: "#007bff"
+  message: "Thank you for visiting."                    # Message to be displayed aside from the button for a few seconds.
+  description: "Support me on Buy me a coffee!"
```
Comment (rribeiro1) [inline docs/_config.yml:83]: Created a different key to set up this integration.

**Review by YoussefRaafatNasry — APPROVED** [review]

**Comment by YoussefRaafatNasry** [comment]
Thanks @rribeiro1 for all your contributions to portfolYOU!

**Pull request merged**